django/djangoproject.com
# Refactored `version-switcher.js`

- Simplified code
- Stopped using jQuery
- Moved refactored code to `djangoproject.js`

This patch should bring no user-facing changes.

Refs #1827

To test this:

1. Navigate to a documentation page
2. Click on a paragraph marker to add a fragment identifier to your current URL (like `#getting-help`)
3. Click on a different documentation version via the floating element
4. Verify that the fragment identifier persisted to the new documentation URL
